other/materials_designer/import_materials_from_standata.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
<p>Should be Install. Imports should be in 2.1.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>